### ISSUE TITLE
parameters_selection or variables_selection

### ISSUE THREAD

**Issue by DominiqueMakowski** [easystats/parameters#51]
Currently the function is named `parameters_selection`, because of consistency with others (e.g. `parameters_standardize`) and because of `packagename_*` convention sometimes used. 

However, this aspect is, I think, more known as "variables selection" (or features selection). 

Same goes for the future `parameters_reduction` that I will soon start to think about. 

Should we change these functions to `variables_*`? Keep both (variables_* as aliases)? Something else?

**Comment by pdwaggoner** [comment]
As far as I think about this, I consider parameters and variables to be related, but distinct. Parameters are the things we are estimating, and variables are the things we think exert some effect on some Y/response. So all variables are the same things as features that are placed in a given specification. And once we fit the model, we get parameter estimates (estimates of the effects) as a result. Thus, the parameters are attached to and derived from the variables/features. E.g., in the term, `$X\beta$`, `X` is the variables/feature and `\beta` is the parameter. 

However, these naming conventions change a little dependent on which field you are talking to. For example, the "variable" (mathematical statistics) versus "feature" (machine learning) distinction. Different names, but same thing. Or same thing for "population" versus "feature space" (which is just the n-dimensional space where all features exist). Not sure if this helps (or is even what you're after here), but it's how I think of it.

**Comment by DominiqueMakowski** [comment]
Yup, I agree that parameters, variables (and features) are different. That's the reason why I am hesitant about the naming of the functions; should be break conventions, add aliases or do nothing

**Comment by pdwaggoner** [comment]
Ah, I see what you're saying. Yeah, this is a tricky one, because on the pkgdown cite for this example, 

```{R}
devtools::install_github("easystats/parameters")
library(parameters)

model <- lm(disp ~ ., data = mtcars)
```

I feel like `parameters_selection(model)` should actually return `model$coefficients`, instead of a vector of the names of the variables/features used to fit the model on RHS (as it currently does, e.g., `> [1] "cyl + hp + wt + qsec + carb"`). 

In my opinion, to be consistent with _statistical_ syntax (even though it would be breaking _easystats_ convention), we should rename `parameters_*` to `variables_*` as you note.

While we could always take the alias route, this may actually confuse less sophisticated researchers who are using the easyverse packages to "make statistics easy", given that parameters are different from variables, but we would effectively be telling them that _either_ is alright. I wouldn't want easystats to be the source of confusion by conflating these terms with the use of aliases.  Just my two cents, but happy to consider any other thoughts here!

**Comment by DominiqueMakowski** [comment]
I agree, but the tricky part, in fact, the current methods for "features selection" actually do a parameters selection rather than a variable selection 😱 (i.e., it runs the, for instance, a stepwise selection with the different parameters, and an initial model with `y ~ x1 + x2 + x3 + x1:x2 + x1:x3 + x2:x3` (with interaction terms) could end up as `y ~ x1 + x2 + x1:x3`, in which case the model has the same variables but different parameters...)

- Basically `parameters_selection` is actually closer to what is done (at least for the current methods) and is consistent with the easyvibe
- `variable_selection`/`feature_selection` is how the concept is often referred to in general.

Maybe let's make it simple and stick to `parameters_*` with tons of keywords in the documentation referring to it as variable/feature selection so that users can easily find it.
